src/services/TestTypesService.ts

```diff
@@ -143,6 +143,7 @@ export class TestTypesService {
 
         const response: any = {
           id: testType.id,
+          suggestedTestTypeIds: testType.suggestedTestTypeIds,
```

Comment:
Need to confirm whether suggestedTestTypeIds are needed in the `getTestTypesById` function. Based on the ticket it doesn't seem we need to do this chage.

Reply:
I have confirmed with Khushboo that this doesn't need to be included. @m-ujaffer